website/templates/institution.mako

```diff
@@ -0,0 +1,52 @@
+<%inherit file="base.mako"/>
+<%def name="title()">Institution</%def>
+
+<%def name="container_class()">container-xxl</%def>
+
+<%def name="content()">
+% if disk_saving_mode:
+    <div class="alert alert-info"><strong>NOTICE: </strong>Forks, registrations, and uploads will be temporarily disabled while the OSF undergoes a hardware upgrade. These features will return shortly. Thank you for your patience.</div>
+% endif
+    <div id="inst">
+<div class="dashboard-header">
+    <div class="row" style="text-align: center">
+        <div class="col-sm-2"></div>
+        <div class="col-sm-3"><img class="img-circle" height="110" width="110" src=${ logo_path }></div>
+        <div class="col-sm-3">
+            <h2>${ name }</h2>
+            <h4><small data-bind="text: instDescription" class="hidden-sm">Loading description ...</small></h4>
```

Comment:
If the institution description is available from the model, can it be shown like this:
```
% if description:
    <h4>${description}</h4>
% endif
```
instead of as a knockout observable? 
Especially since  "Loading description..." is shown even when the institution has no description.